### ISSUE TITLE
Incorrect checksum format in paxkage index JSON

### ISSUE THREAD

**Issue by jgromes** [MCUdude/MegaCoreX#154]
Hi,

when running a CI on one of my projects, I'm getting the following error when attempting to install MegaCoreX using arduino-cli:

```
Installing arduino:avr-gcc@7.3.0-atmel3.6.1-arduino7...
arduino:avr-gcc@7.3.0-atmel3.6.1-arduino7 installed
Error during install: Cannot install tool MegaCoreX:avrdude@7.0.0-arduino2: testing local archive integrity: testing archive checksum: invalid checksum format: 03f29bae3b53ac1a0e469c1e79f1696a572d7be632838484526674501de0248e
```

Looking at the package index JSON file, I noticed that one of the checksums format is different:

```
{
              "size": "617568",
              "checksum": "SHA-256:ff0b5274182ffacbdea72cab24cb74e8f4b81bc654a4539bbcaa5445825b5b6d",
              "host": "x86_64-apple-darwin12",
              "archiveFileName": "avrdude_7.0-arduino.2_macOS_64bit.tar.gz",
              "url": "http://downloads.arduino.cc/tools/avrdude_7.0-arduino.2_macOS_64bit.tar.gz"
            },
            {
              "size": "989218",
              "checksum": "03f29bae3b53ac1a0e469c1e79f1696a572d7be632838484526674501de0248e",
              "host": "x86_64-linux-gnu",
              "archiveFileName": "avrdude_7.0-arduino.2_Linux_64bit.tar.gz",
              "url": "http://downloads.arduino.cc/tools/avrdude_7.0-arduino.2_Linux_64bit.tar.gz"
            },
            {
              "size": "1002537",
              "checksum": "SHA-256:97ed291b15dc421124c279ab9bf25d8a30e82d495ef2b4c41febb7f2d8f59c01",
              "host": "i686-linux-gnu",
              "archiveFileName": "avrdude_7.0-arduino.2_Linux_32bit.tar.gz",
              "url": "http://downloads.arduino.cc/tools/avrdude_7.0-arduino.2_Linux_32bit.tar.gz"
            },
```

For `x86_64-linux-gnu`, the "SHA-256:" prefix seems to be missing, is that intended?

**Comment by MCUdude** [comment]
Thank for reporting! The missing `SHA-256:` is absolutely the issue. I've fixed this in [1d4bebc](https://github.com/MCUdude/MegaCoreX/commit/1d4bebc755392f0e7fb2b5749adb5645c728ab92). Please delete the downloaded JSON boards manager file and try again

**Comment by jgromes** [comment]
```
```
